.github/workflows/wheels-manylinux-test.yml

```diff
@@ -66,13 +66,19 @@ jobs:
           {
             "pull-request": [
               { "arch": "amd64", "python": "3.8", "ctk": "11.8.0", "image": "ubuntu18.04", "test-type": "unit", "test-command": "${{ inputs.test-unittest }}", "gpu": "v100", "driver": "520"  },
-              { "arch": "arm64", "python": "3.8", "ctk": "11.8.0", "image": "ubuntu20.04", "test-type": "smoke", "test-command": "${{ inputs.test-smoketest }}", "gpu": "a100", "driver": "520"  }
+              { "arch": "arm64", "python": "3.8", "ctk": "11.8.0", "image": "ubuntu20.04", "test-type": "smoke", "test-command": "${{ inputs.test-smoketest }}", "gpu": "a100", "driver": "525"  },
```

Comment:
Was bumping some of the 11.8 runs to driver 525 intentional?

Reply:
Yes, ARM instances are limited so we will only have 525 runners.

Reply:
Got it.